.github/workflows/radar-release-actions.yml

```
@@ -0,0 +1,18 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [created]
```

We've been using `published` instead of `created` here on the Android and iOS GitHub Actions Triggers. May be good to use the same, for consistent behavior when cutting releases.

```suggestion
    types: [ published ]
```